### PR TITLE
[INFRA-1212] Rename branch to main

### PR DESCRIPTION
**Pull request by mariechatfield**

### CODE REVIEW

**Comment by mariechatfield** [comment]
Build is failing due to outdated ssh key; getting that updated now

File: .circleci/config.yml [hunk 1]
```diff
@@ -101,12 +101,12 @@ workflows:
           filters:
             branches:
               ignore:
-                - dw-master
+                - dw-main
 
   build-test-deploy:
     jobs:
       - master_build:
```
Comment (chrisbodhi) [inline .circleci/config.yml]: Should this job name be changed, too?

File: .circleci/release.sh [hunk 2]
```diff
@@ -11,7 +11,7 @@ CIRCLE_TOKEN=${1:-}
 
 ACCOUNT=datadotworld
 PROJECT=aws-sdk-java
-BRANCH=dw-master
+BRANCH=dw-main
```
Comment (chrisbodhi) [inline .circleci/release.sh]: Why not take this opportunity to standardize on `main`?

Comment (mariechatfield) [reply .circleci/release.sh]: I think it was to differentiate between the main branch of the original repo, but I suppose it doesn't hurt anything

**Pull request merged**